samples/java_layout/src/main/java/io/customer/android/sample/java_layout/sdk/CustomerIORepository.java

```diff
@@ -103,11 +103,11 @@ private void configureSdk(CustomerIO.Builder builder, final CustomerIOSDKConfig
     }
 
     public void identify(@NonNull String email, @NonNull Map<String, String> attributes) {
-        CustomerIO.instance().identify(email, attributes);
```

Comment:
Due to this removal, apps may no longer send push token and events to Fly as profile will no longer be identified to older SDK. But I think its okay as it'll keep the logs clean and easier to remove old code later.